src/matchmaker/drawing.js

```diff
@@ -0,0 +1,43 @@
+const {
+  separateTopFromBottomPlayers,
+  pairPlayersRoundOne
+} = require("./drawingAlgorithms.js");
+
+const { createMatches } = require("./match.js");
+
+// drawRound : [players] -> [matches]
+function drawRound({ players }) {
+  const isFirstRound = players[0].matchIds.length == 0;
+
+  if (!isFirstRound) {
```

Comment:
it seems more clear when you don't negate it. Also you could simplify it by using a ternary expression for getting the matches and then just returning them, like so: `const pairings = players[0].matchIds.length === 0 ? drawFirstRound(players) : drawLaterRound(players); return createMatches(pairings);` (prettier should then move this to multiple lines)

Reply:
good point